### PR TITLE
Allow user to make yamllint rule mandatory

### PR DESCRIPTION
**Pull request by ssbarnea**
Closes: #1538

### CODE REVIEW

File: src/ansiblelint/rules/YamllintRule.py [hunk 1]
```diff
@@ -38,6 +43,10 @@
 Specific tag identifiers that are printed at the end of rule name,
 like 'trailing-spaces' or 'indentation' can also be be skipped, allowing
 you to have a more fine control.
+
+By default this rule is not used hwne yamllint library is missing. If you want
```
Comment (dosmanak) [inline src/ansiblelint/rules/YamllintRule.py:47]: Typo "hwne" => "when"

**Comment by dosmanak** [comment]
This change does not solve the non-deterministic behaviour on yamllint presence in the system :-(

**Comment by ssbarnea** [comment]
@dosmanak It does, if you add the now documented rule. On the other hand, if you want deterministic results, you could opt for using a container. Different results were caused by different environments, that was not really a bug in the linter.

**Comment by dosmanak** [comment]
I still do not understand the default behaviour.
When the yamllint is present it is used, whilst if missing it fails silently. Why have you chose such illogical approach? Perhaps I am the only one who feels that way and I will not push the discussion further.

**Pull request merged**